src/pages/create/TypeStep.tsx

```diff
@@ -21,22 +22,37 @@ const TypeStep = <TFormData extends FormDataIntersection>({
       render={({ field }) => {
         return (
           <Inline
-            spacing={5}
+            spacing={{
+              default: 6,
+              m: 4,
+            }}
             alignItems="center"
-            justifyContent="space-between"
+            justifyContent="center"
```

Comment:
Looks good!! 💅 
I think I just prefer the layout with `space-between` when the boxes are aligned with the left and right margins...

Reply:
I also find it visually more pleasing, but the buttons are so far apart to click on a big screen